### PR TITLE
EDU-11369 - Experimental Exports - Hooks & Components

### PR DESCRIPTION
**Pull request by Mariana-Caetano**
#### Types of changes
- [x] New content (guides, endpoints, app documentation)
- [ ] Improvement (make a documentation even better)
- [ ] Fix (fix a documentation error)
- [ ] Spelling and grammar accuracy (self-explanatory)

Atualizando a sidebar: https://github.com/vtexdocs/devportal/pull/804

### CODE REVIEW

File: docs/faststore/docs/customization/building-sections/experimental-exports-hooks-and-components.mdx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+4. Save your changes.
+5. Open a terminal and run `yarn dev` to test the implementation.
+
+## List of hooks and components
+
+Explore the features of the experimental hooks and the `Image_unstable` component.
+
+- [Hooks](#hooks)
+- [`Image_unstable`](#image)
```
Comment (Mariana-Caetano) [inline docs/faststore/docs/customization/building-sections/experimental-exports-hooks-and-components.mdx:52]:
```suggestion
- [`Image_unstable`](#image_unstable-component)
```

File: docs/faststore/docs/customization/building-sections/experimental-exports-hooks-and-components.mdx [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+## Instructions
+
+1. Open your store project in a code editor, then open the component or section file where you want to import the hook or component.
+2. Declare the hook or component into your custom component file.
+
+    ```tsx
+    import { use{ComponentName}_unstable as use{ComponentName} } from '@faststore/core/experimental'
+    ```
+
```
Comment (carolinamenezes) [inline docs/faststore/docs/customization/building-sections/experimental-exports-hooks-and-components.mdx:21]: Acho válido incluir um disclaimer indicando que {ComponentName} deve ser substituído pelo nome do componente

File: docs/faststore/docs/customization/building-sections/experimental-exports-hooks-and-components.mdx [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+For example, you can create a custom [BuyButton](https://developers.vtex.com/docs/guides/faststore/molecules-buy-button) that adds products to the cart and displays a personalized message based on user preferences or order history.
+
+## Before you begin
+
+All experimental [hooks and components](#list-of-hooks-and-components) are suffixed with `_unstable`, indicating that they are subject to change and may not be stable for use in production.
+
+## Instructions
+
+1. Open your store project in a code editor, then open the component or section file where you want to import the hook or component.
+2. Declare the hook or component into your custom component file.
```
Comment (carolinamenezes) [inline docs/faststore/docs/customization/building-sections/experimental-exports-hooks-and-components.mdx:16]:
```suggestion
2. Import the hook or component into your custom component file.
```

File: docs/faststore/docs/customization/building-sections/experimental-exports-hooks-and-components.mdx (hunk 2 above)
Comment (Mariana-Caetano) [reply docs/faststore/docs/customization/building-sections/experimental-exports-hooks-and-components.mdx]: Boa! Adicionei na [linha 22](https://github.com/vtexdocs/dev-portal-content/pull/1438/files#diff-1046ded8bcdc13137a7ab27d66a8bbcb54139c2ee47699e2dba29d7fbcce74a0R22).

**Review by carolinamenezes — APPROVED** [review]

**Pull request merged**